code/HabitTracker/app/src/main/java/com/example/habittracker/User.java

```diff
@@ -2,13 +2,20 @@
 
 import android.util.Log;
 
-public class User {
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.tasks.OnCompleteListener;
```

Comment:
Can you remove these unnecessary imports from the User class before merging?

Reply:
Fixed